satpy/readers/viirs_compact.py

```diff
@@ -16,6 +16,10 @@
 # You should have received a copy of the GNU General Public License along with
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
 """Compact viirs format.
+
+.. note:: It should be possible to further enhance this reader by performing the
```

Comment:
Does this render properly. Does the content need to be on a separate line (with a blank line in between)?

Reply:
Not according to the RST doc.
https://livesphinx.herokuapp.com/ agrees